### PR TITLE
Use "dotnet test --verbosity" arg for console verbosity

### PR DESCRIPTION
**Pull request by Faizan2304**
1) Add support to use "dotnet test --verbosity" args for console verbosity
2) Remove invalid help message

### CODE REVIEW

File: src/Microsoft.TestPlatform.Build/Tasks/VSTestTask.cs [hunk 1]
```diff
@@ -181,6 +187,25 @@ internal IEnumerable<string> CreateArgument()
                 }
             }
 
+            if (!string.IsNullOrWhiteSpace(this.VSTestVerbosity) &&
+                (string.IsNullOrEmpty(this.VSTestLogger) || this.VSTestLogger.IndexOf("console", StringComparison.OrdinalIgnoreCase) < 0))
```
Comment (smadala) [inline src/Microsoft.TestPlatform.Build/Tasks/VSTestTask.cs]: use `this.VSTestLogger.Startwith()`.

File: src/Microsoft.TestPlatform.Build/Tasks/VSTestTask.cs [hunk 2]
```diff
@@ -181,6 +187,25 @@ internal IEnumerable<string> CreateArgument()
...
+            {
+                var normalTestLogging = new List<string>() { "n", "normal", "d", "detailed", "diag", "diagnostic" };
+                var minimalTestLogging = new List<string>() { "m", "minimal" };
+
+                string vsTestVerbosity = "quiet";
```
Comment (singhsarab) [inline src/Microsoft.TestPlatform.Build/Tasks/VSTestTask.cs]: default should be minimal

File: src/Microsoft.TestPlatform.Build/Tasks/VSTestTask.cs [hunk 3]
```diff
@@ -181,6 +187,25 @@ internal IEnumerable<string> CreateArgument()
                 }
             }
 
+            if (!string.IsNullOrWhiteSpace(this.VSTestVerbosity) &&
+                (string.IsNullOrEmpty(this.VSTestLogger) || this.VSTestLogger.IndexOf("console", StringComparison.OrdinalIgnoreCase) < 0))
+            {
+                var normalTestLogging = new List<string>() { "n", "normal", "d", "detailed", "diag", "diagnostic" };
```
Comment (singhsarab) [inline src/Microsoft.TestPlatform.Build/Tasks/VSTestTask.cs]: Add tests for each mapping.

**Review by singhsarab — APPROVED** [review]
LGTM with some suggestions.

**Review by smadala — APPROVED** [review]

File: src/Microsoft.TestPlatform.Build/Tasks/VSTestTask.cs (hunk 2 above)
Comment (Faizan2304) [inline src/Microsoft.TestPlatform.Build/Tasks/VSTestTask.cs]: Fixed!!

File: src/Microsoft.TestPlatform.Build/Tasks/VSTestTask.cs (hunk 1 above)
Comment (Faizan2304) [inline src/Microsoft.TestPlatform.Build/Tasks/VSTestTask.cs]: Fixed in next commit.

File: src/Microsoft.TestPlatform.Build/Tasks/VSTestTask.cs (hunk 3 above)
Comment (Faizan2304) [inline src/Microsoft.TestPlatform.Build/Tasks/VSTestTask.cs]: Fixed in next commit.

**Pull request merged**